dbms/src/Flash/tests/gtest_split_tasks.cpp

```diff
@@ -95,15 +95,15 @@ try
 
     size_t task_size = tasks.size();
     std::vector<String> executors = {
-        "exchange_sender_6 | type:Hash, {<0, String>}\n"
-        " table_scan_1 | {<0, String>}",
+        "exchange_sender_6 | type:Hash, {<0, String>, <1, String>}\n"
```

Comment:
because we can distinguish the column name by table_name and column_name, so the columnPrune result is different.